sdk/honcho/client.py

```diff
@@ -59,54 +69,68 @@ def create_session(self, user_id: str, location_id: str = "default", session_dat
             session_data (Dict, optional): Optional session metadata
 
         Returns:
-            Dict: The Session object of the new Session
+            Dict: The Session object of the new Session`
 
         """
-        data = {
-                "location_id": location_id,
-                "session_data": session_data
-                }
+        data = {"location_id": location_id, "session_data": session_data}
         url = f"{self.base_url}/users/{user_id}/sessions"
         response = requests.post(url, json=data)
-        return response.json()
-
-    def update_session(self, user_id: str, session_id: int, session_data: Dict):
-        """Update the metadata of a session
-
-        Args:
-            user_id (str): The User ID representing the user, managed by the user
-            session_id (int): The ID of the Session to update
-            session_data (Dict): The Session object containing any new metadata
-
-        Returns:
-            Dict: The Session object of the updated Session
-
-        """
-        data = {
-                "session_data": session_data
-                }
-        url = f"{self.base_url}/users/{user_id}/sessions/{session_id}"
-        response = requests.put(url, json=data)
-        return response.json()
-
-    def delete_session(self, user_id: str, session_id: int):
-        """Delete a session by marking it as inactive
+        data = response.json()
+        return Session(
+            self,
+            id=data["id"],
+            user_id=user_id,
+            location_id=location_id,
+            session_data=session_data,
+            is_active=data["is_active"],
+        )
+
+
+class Session:
+    def __init__(
+        self,
+        client: Client,
+        id: int,
+        user_id: str,
+        location_id: str,
+        session_data: dict | str,
+        is_active: bool,
```

Comment:
`is_active` should probably be a read only value or something that is asserted by checking against the api. 

If a developer manually sets the session to active via the object then attempts to add a message it will cause an unhandled error. Can either make it self corrected in the catch statement for the error or let it crash. 

Either way should throw an error